src/OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -9,10 +9,11 @@
 #include <absl/meta/type_traits.h>
 #include <absl/strings/str_format.h>
 #include <pthread.h>
-#include <stddef.h>
```

Comment:
The reason `<stddef.h>` is used instead of `<cstddef>` is that we use `size_t` and not `std::size_t` and such. I think you are going against IWYU by changing this.

Reply:
reverted